docs/website/docs/user-guides/advanced/connecting-a-service.md

```diff
@@ -0,0 +1,226 @@
+---
+title: Connecting to a service
+sidebar_position: 1
+---
+
+This tutorial will show you how you can connect your Go application to a mongodb service.
+
+Building on top of the [Go quickstart guide](../quickstart/go.md), we will extend the application to check if it is connected to a mongodb service.
+
+### Pre-requisites
+1. Install Service Binding Operator on your cluster.  See [installation instruction](https://operatorhub.io/operator/service-binding-operator).
+
+2. Install Percona Server Mongodb operator on your cluster. See [installation instruction](https://operatorhub.io/operator/percona-server-mongodb-operator).
+:::note
+The operator will be installed in a new namespace called "my-percona-server-mongodb-operator" and for the sake of simplicity, we will use this namespace for this guide. 
```

Comment:
> for the sake of simplicity, we will use this namespace for this guide.
>
How are we using this namespace in this guide? This note is not necessary to me, as we are installing the service instance in the current namespace, and will be using it from there, no?

Reply:
Perhaps you're right. It's not a necessary note.